Zend/tests/gh13670_001.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+GH-13670 001
+--FILE--
+<?php
+
+register_shutdown_function(function () {
+    global $shutdown;
+    $shutdown = true;
+});
+
+class Cycle {
+    public $self;
+    public function __construct() {
+        $this->self = $this;
+    }
+    public function __destruct() {
+        global $shutdown;
+        if (!$shutdown) {
+            new Cycle();
+        }
+    }
+}
+
+for ($i = 0; $i < 10001; $i++) {
```

Comment:
Can these magical numbers be defined using shared consts for all GC tests?

Reply:
Good point. I've updated the test to use gc_status()['threshold'] to fetch the default threshold.